### PR TITLE
Provide note on NuGet package to install for enrichers

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide note on NuGet package to install for enrichers.

### CODE REVIEW

File: docs/features/telemetry-enrichment.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ layout: default
 
 # Telemetry Enrichment
 
+## Installation
+
+This feature requires to install our NuGet package
+
+```shell
+PM > Install-Package Arcus.Observability.Telemetry.Serilog
```
Comment (tomkerkhove) [inline docs/features/telemetry-enrichment.md]: This makes me wonder if we shouldn't rename it to `Arcus.Observability.Telemetry.Serilog.Enrichers`

Comment (stijnmoreels) [reply docs/features/telemetry-enrichment.md]: Could be yeah. On the other hand; will we ever enrich Serilog sinks or other stuff to have another namespace for `...Telemetry.Serilog.XXX`?

Comment (tomkerkhove) [reply docs/features/telemetry-enrichment.md]: Yes, see #35 already introduces an emptry project

Comment (stijnmoreels) [reply docs/features/telemetry-enrichment.md]: Ok, then definitly moving to `...Enrichers`! I guess 🏭

Comment (tomkerkhove) [reply docs/features/telemetry-enrichment.md]: https://github.com/arcus-azure/arcus.observability/issues/36

**Pull request merged**